accounts/views/profile.py

```diff
@@ -0,0 +1,62 @@
+from django.http import Http404
+from django.contrib.auth.decorators import login_required
+from django.urls import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.views.generic import UpdateView
+
+from accounts.forms import EmployeeEditProfile, EmployerEditProfile
+
+from accounts.models import EmployeeProfile, EmployerProfile
+
+
+class UpdateEmployeeProfileView(UpdateView):
+    """
+    Update employee/jobseeker profile info
+    """
+    model = EmployeeProfile
+    form_class = EmployeeEditProfile
+    context_object_name = 'employee'
+    template_name = 'accounts/profile/employee/../../templates/accounts/profile/edit-profile.html'
+    success_url = reverse_lazy('accounts:employee-profile-update')
+
+    @method_decorator(login_required(login_url=reverse_lazy('accounts:login')))
```

Comment:
Can any logged in user access it? We need to apply the role based permissions too